src/scenes/SearchResults/search.graphql

```diff
@@ -0,0 +1,42 @@
+query Search($input: SearchInput!) {
+  search(input: $input) {
+    items {
+      ... on Country {
+        ...DisplayCountry
+      }
```

Comment:
We actually just figured out we don't need to specify the type here to put the fragment inside. Because the fragment defines what it is on.
```suggestion
      ...DisplayCountry
```

You could think of it like a named vs anonymous function.
```gql
... on X {}
```
```gql
myX on X {}
```

Reply:
I'll change this to the new way have just been trying to deal with issue with url being redirected from hook.